lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -90,22 +91,42 @@ sub _get_ptero_execute_method {
 
     my $self = shift;
     my $log_dir = shift;
+    my $ptero_lsf_parameters = $self->_get_ptero_lsf_parameters();
+    $ptero_lsf_parameters->{command} = sprintf(
+        'genome ptero wrapper --command-class %s '
+        .'--method execute --log-directory %s',
+        $self->command, $log_dir);
```

Comment:
Some layer above PTero would have to be responsible for crafting the appropriate command for submission to LSF, especially if a log directory contained spaces or other characters that required quoting.